djangocms_pageadmin/static/djangocms_pageadmin/css/actions.css

```diff
@@ -19,9 +19,55 @@ a.btn.cms-page-admin-action-btn {
     -webkit-box-sizing: border-box;
             box-sizing: border-box;
 }
+
+ul.cms-pagetree-dropdown-menu-inner li {
+
+    font-family:"Helvetica Neue",Helvetica,Arial,sans-serif !important;
+    list-style-type: none;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    padding: 2px 6px;
+}
+
+ul.cms-pagetree-dropdown-menu-inner li:hover {
+    list-style-type: none;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    background-color: #0bf;
+}
+
+ul.cms-pagetree-dropdown-menu-inner li:hover a.btn {
+    color: #fff !important;
```

Comment:
No other way that to use important everywhere?

Reply:
Sadly no.

Reply:
Well. Maybe there is but it's a lot more work. Currently I'm just collecting the buttons and then using CSS to reformat them so that they appear as part of the burger menu.

For me to avoid the `!important` bits I would have to refactor the javascript, so that instead of just moving the actions, it builds up new elements based on the attributes of those actions. Do you think it's worth it?